sendgrid/helpers/permissions/unit_tests.py

```diff
@@ -0,0 +1,437 @@
+from permissions import Permissions
+
```

Comment:
This project's unit tests are run with the [`unittest`](https://docs.python.org/3/library/unittest.html#basic-example) framework, which looks for a subclass of `unittest.TestCase` and runs all its the methods starting with `test`.  Take a look at some of the [existing tests](https://github.com/sendgrid/sendgrid-python/blob/master/test/test_mail.py) to see the format they're in.